packetinjector/server.go

```diff
@@ -92,8 +92,8 @@ func (o *onDemandPacketInjectServer) CreateTask(srcNode *graph.Node, resource ty
 
 	metadata := &InjectionMetadata{
 		PacketInjectionRequest: *pp,
-		ID:    pp.UUID,
-		State: "active",
+		ID:                     pp.UUID,
```

Comment:
don't break the idents

Reply:
changed from go 1.10 to go 1.11: https://github.com/golang/go/issues/26098